engine/app/controllers/good_job/assets_controller.rb

```diff
@@ -35,11 +35,11 @@ def rails_ujs_js
     end
```

Comment:
I'm not sure if these practices for the engine's assets also apply to the _vendored_ assets. Perhaps these should also be namespaced (see the lines above this comment)?